### PR TITLE
tests/sce-in-ds: Add to Makefile.am

### PR DESCRIPTION
**Pull request by ybznek**
I forgot to append test to 
Append forgotten test

### CODE REVIEW

**Comment by mpreisler** [comment]
Why is this going to TESTS and not EXTRA_DIST?

**Comment by ybznek** [comment]
I think, only one line is missing (see whole file).

File: tests/sce/Makefile.am [hunk 1]
```diff
@@ -9,7 +9,8 @@ TESTS_ENVIRONMENT= \
 
 TESTS = test_sce.sh \
 		test_passing_vars.sh \
-		test_check_engine_results.sh
+		test_check_engine_results.sh \
+		test_sce_in_ds.xml
```
Comment (mpreisler) [inline tests/sce/Makefile.am]: .xml ?!

**Comment by jan-cerny** [comment]
@ybznek test_sce_in_ds.xml should be in extra_dist
Please verify that you can successfully run and pass `make distcheck` with your changes.
The `make distcheck` is different than `make check`.

**Comment by ybznek** [comment]
@jan-cerny 
https://github.com/ybznek/openscap/blob/maint-1.0-fix-tests/tests/sce/Makefile.am#L28
Isn't it what you want?

**Comment by mpreisler** [comment]
@jan-cerny It is in EXTRA_DIST and it was there already.

Will merge this after Jenkins finishes.

**Comment by jan-cerny** [comment]
@mpreisler Yes, I see it, thanks.

**Pull request merged**